cryptographic_estimators/UOVEstimator/UOVAlgorithms/kipnis_shamir.py

```diff
@@ -23,26 +23,23 @@
 
 
 class KipnisShamir(UOVAlgorithm):
-    """
-    Construct an instance of Kipnis-Shamir estimator
-
-    in []_, Kipnis and Shamir proposed a powerful attack against the balanced Oil
-    and Vinegar signature scheme (n = 2v), which finds an equivalent private key in
-    polynomial time. This key can then be used to generate signatures for arbitrary
-    messages.
-
-    INPUT:
-
-    - ``problem`` -- an instance of the UOVProblem class
-    - ``memory_access`` -- specifies the memory access cost model (default: 0, choices: 0 - constant, 1 - logarithmic, 2 - square-root, 3 - cube-root or deploy custom function which takes as input the logarithm of the total memory usage)
-    - ``complexity_type`` -- complexity type to consider (0: estimate, 1: tilde O complexity, default: 0)
-
-    EXAMPLES::
-
-
-    """
-
     def __init__(self, problem: UOVProblem, **kwargs):
+        """Construct an instance of Kipnis-Shamir estimator.
+
+        In [Beu20]_, Kipnis and Shamir proposed a powerful attack against the balanced Oil
+        and Vinegar signature scheme (n = 2v), which finds an equivalent private key in
+        polynomial time. This key can then be used to generate signatures for arbitrary
+        messages.
+
+        Args:
+            problem (UOVProblem): An instance of the UOVProblem class.
+            **kwargs: Additional keyword arguments.
+                memory_access: Specifies the memory access cost model (default: 0, choices: 
+                    0 - constant, 1 - logarithmic, 2 - square-root, 3 - cube-root or deploy 
+                    custom function which takes as input the logarithm of the total memory usage).
+                complexity_type: Complexity type to consider (0: estimate, 1: tilde O complexity, default: 0).
```

Comment:
wrong indentation

Reply:
to be fixed in  #198

Reply:
Nothing to fix. Indentation is correctly parsed by our custom Sphinx extension (see https://github.com/Crypto-TII/CryptographicEstimators/pull/211).